crates/torii/cli/src/options.rs

```diff
@@ -90,6 +90,10 @@ impl Default for RelayOptions {
 #[derive(Debug, clap::Args, Clone, Serialize, Deserialize, PartialEq)]
 #[command(next_help_heading = "Indexing options")]
 pub struct IndexingOptions {
+    /// If indexing should be enabled
+    #[arg(long, default_value_t = true, help = "If indexing should be enabled.")]
+    pub enabled: bool,
```

Comment:
```suggestion
    /// If indexing should be enabled
    #[arg(long, default_value_t = true, help = "If indexing should be enabled.")]
    #[serde(default)]
    pub enabled: bool,
```
Add serde default to ensure it's not required to be given in configuration files.